primitives/runtime/src/traits.rs

```diff
@@ -156,6 +156,12 @@ pub trait EnsureOrigin<OuterOrigin> {
 	}
 	/// Perform the origin check.
 	fn try_origin(o: OuterOrigin) -> result::Result<Self::Success, OuterOrigin>;
+
+	/// Returns an outer origin capable of passing `try_origin` check.
+	/// 
+	/// ** Should be used for benchmarking only!!! **
```

Comment:
I won't complain because I don't know an alternative which is better :D mocking this in a way that has less footprint would be good. But I read in the discussion that this is apparently as good as it gets so 👍